### ISSUE TITLE
[a11y]: ContentSwitcher [icon only & icon only with layer] - screen reader doesn't identify icons

### ISSUE THREAD

**Issue by andreancardona** [carbon-design-system/carbon#13979]
### Package

@carbon/react

### Browser
Safari & FF & iPhone

### Package version

carbon/react@1.30.0

### Automated testing tool and ruleset
Manual QA

### Assistive technology
JAWS & VO

### Description

Screen reader does not identify the Icons present in the component

Screen reader does not identify the Icons present in the component. Hence the user would not know their presence.

### WCAG 2.1 Violation
n/a

### Reproduction/example
Record `Number 49` in the below table:
https://airtable.com/shrFsaeHhImOkuVNR

### Steps to reproduce

If these icons convey meaningful information, these should be made accessible to the screen reader. Screen reader should identify the image role with appropriate alt text that describes the purpose of each image. 

This needs to be confirmed with the team 

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by andreancardona** [comment]
Just chatted with Gower that confirmed on JAWS this would be redundant label and it is not needed - closing the issue.
